src/GlobalStyle.tsx

```diff
@@ -10,15 +10,11 @@ const GlobalStyle = createGlobalStyle`
     -webkit-overflow-scrolling: touch;
     scrollbar-width: thin;
     scrollbar-color: ${theme.colors.athensGray} transparent;
-    &::-webkit-scrollbar {
```

Comment:
this changes the style of all the scrollbars. im not sure you wanted to do that

Reply:
you could remove the `scroll` from where i commented

Reply:
it was causing issues in other parts of the site e.g. conscious shopping so will leave it out for now